test/extensions/filters/common/ext_authz/ext_authz_grpc_impl_test.cc

```diff
@@ -90,7 +90,8 @@ TEST_F(ExtAuthzGrpcClientTest, AuthorizationOk) {
 
   envoy::service::auth::v3::CheckRequest request;
   expectCallSend(request);
-  client_->check(request_callbacks_, request, Tracing::NullSpan::instance(), stream_info_);
+  client_->check(request_callbacks_, request, Tracing::NullSpan::instance(),
+                 *Http::StaticEmptyHeaders::get().request_headers, stream_info_);
```

Comment:
There are many similar changes. And these changes make no sense for now. I mean, we can delay these to the future dedicated PR.

Reply:
Reverted check to the original form, so all those changes were also reverted.